.gitignore

```diff
@@ -9,4 +10,4 @@ downloads/
 *.pdb
 *.dump
 *.log
-*.a
\ No newline at end of file
+*.a
```

Comment:
Is there a white space change here?

Reply:
Yes, the end-of-file empty line was missing.